db/schema.rb

```diff
@@ -30,6 +30,16 @@
     t.datetime "updated_at"
   end
 
+  create_table "recommended_links", force: :cascade do |t|
+    t.string  "title",        limit: 255
+    t.string  "link",         limit: 255
+    t.string  "description",  limit: 255
+    t.string  "keywords",     limit: 255
+    t.string  "search_index", limit: 255
+    t.text    "comment",      limit: 65535
+    t.integer "user_id",      limit: 4
```

Comment:
Yup, this was a simple integer which defaulted to length 4. I have changed this to a reference to the `users` table and also added a migration for the `bets` table to do the same. Hence the new length in the database is 11 (same as primary key on the `users` table), although for some reason the `schema.rb` still says 4...